app/src/main/java/com/cornellappdev/volume/data/repositories/PublicationRepository.kt

```diff
@@ -23,17 +22,18 @@ class PublicationRepository @Inject constructor(private val networkApi: NetworkA
     suspend fun fetchAllPublications(): List<Publication> =
         networkApi.fetchAllPublications().dataAssertNoErrors.mapDataToPublications()
 
-    suspend fun fetchPublicationsByIDs(pubIDs: List<String>): List<Publication> =
-        networkApi.fetchPublicationsByIDs(pubIDs).dataAssertNoErrors.mapDataToPublications()
+    suspend fun fetchPublicationsByIDs(slugs: List<String>): List<Publication> =
```

Comment:
same thing here

Reply:
Oops forgot to change the name, thank you!